### ISSUE TITLE
wrong warning for isEmpty rule

### ISSUE THREAD

**Issue by amrangry** [realm/SwiftLint#2934]
When i use the below code:

```
guard  count  > 0 else  {
   return
}
```

BTW count is Int

I got the following issue 

> Empty Count Violation: Prefer checking `isEmpty` over comparing `count` to zero. (empty_count)

**Comment by marcelofabri** [comment]
Duplicate of #827
